src/app/view/view-project/issue/issue.component.ts

```diff
@@ -70,16 +71,16 @@ export class AddIssueComponent {
   checkpointName = new FormControl('', [Validators.required]);
   issueText = new FormControl('', [Validators.required]);
   checkpoints = [
-    'Design',
-    'Ideation',
-    'Market',
-    'Tech Stack',
-    'Product Domain',
-    'MVP',
-    'Security',
-    'Funding',
-    'Team',
-    'Miscellaneous'
+    { name: 'Design' },
```

Comment:
What exactly is the need to convert the array into the object?

Reply:
Previously i was trying to access checkpoint name through this.